apps/voting/contracts/Voting.sol

```diff
@@ -58,13 +58,15 @@ contract Voting is IForwarder, AragonApp {
     // We are mimicing an array, we use a mapping instead to make app upgrade more graceful
     mapping (uint256 => Vote) internal votes;
     uint256 public votesLength;
+    uint64 public objectionTime;
 
     event StartVote(uint256 indexed voteId, address indexed creator, string metadata);
     event CastVote(uint256 indexed voteId, address indexed voter, bool supports, uint256 stake);
```

Comment:
I am not sure, whether it's enough or not, to have only the `CastVote` event for both voting phases. 
Maybe it's better to emit smth like `ObjectVote` during the objection phase? 

Could be helpful for the external monitoring and indexers.

WDYT?

Reply:
Good point. Added  `CastObjection` event during the objection phase.